lib/components/related_artists/index.js

```diff
@@ -16,24 +17,30 @@ class RelatedArtists extends React.Component {
 
   constructor(props) {
     super(props)
-    this.state = this.layoutState()
+    this.state = {
+        columns: 0,
+        imageSize: {
+          width: 1,
+          height: 1,
+        },
+    }
   }
 
-  layoutState() : Object {
-    const width = Dimensions.get('window').width
-    const isPad = width > 700
-    const isPadHorizontal = width > 1000
+  layoutState(currentLayout) : Object {
+    const width = currentLayout.width
+    const isPad = width > 600
+    const isPadHorizontal = width > 900
 
     // TODO: Document what these margins are based on.
     let columnCount
     let margins
     if (isPad) {
       if (isPadHorizontal) {
         columnCount = 4
-        margins = 140
+        margins = 90
```

Comment:
I really want to make these calculated the way they are [here](https://github.com/artsy/emission/blob/master/lib/components/artwork_grids/generic_grid.js#L38), so that's why this is still a WIP.